examples/asr/speech_to_text_eval.py

```diff
@@ -154,6 +177,52 @@ def main(cfg: EvaluationConfig):
             f"contain value for `pred_text`."
         )
 
+    if cfg.use_per:
+        per_data_obj = PERData(
+            hypotheses=predicted_text,
+            references=ground_truth_text,
+            punctuation_marks=list(cfg.text_processing.punctuation_marks),
```

Comment:
Which config is this ? This file doesn't have text_processing.xyz

Reply:
It referred to this - https://github.com/ssh-meister/NeMo/blob/per/examples/asr/speech_to_text_eval.py#L106